packages/fast-usdc/test/supports.ts

```diff
@@ -229,11 +229,7 @@ export const commonSetup = async (t: ExecutionContext<any>) => {
       timer,
       localchain,
       cosmosInterchainService,
-      // TODO remove; bootstrap doesn't have a zone
```

Comment:
I dunno if this is quite relevant, but...

The bootstrap vat does in fact have and use a zone that, for example, core eval stuff can be permitted to use.

https://github.com/Agoric/agoric-sdk/blob/6498ae1ecd1e8ada0eec29d2714e8cf69efbcee9/packages/vats/src/core/boot-chain.js#L38

Reply:
Good point. This one was actually the `contractZone` but if fast-usdc testing ever needs the actual root zone it should go here in bootstrap.